### PR TITLE
german cheat sheet final touches (picks up #1368)

### PR DESCRIPTION
**Pull request by seisfeld**
So here we are, let's refine the german cheat sheet finally. I added everything from #1368 as suggested there, changed the order in one section and also changed some minor things i thought looked odd.

Question for the native english folks: In the ```Etiquette``` section, it was suggested to spell "night" lower case. But "Evening" and "Morning" is upper case. Please let me know is that is supposed to be like this. In general it would be good if both native german and native english folks can comment if this is ok now. If not please comment and i'll make the necessary changes. :+1: Maybe we can nail this one once and for all. :wink: 

/cc @moollaza

### CODE REVIEW

**Comment by moollaza** [comment]
>Question for the native english folks: In the Etiquette section, it was suggested to spell "night" lower case. But "Evening" and "Morning" is upper case.

In general "evening", "morning" and "night" are always spelled with lowercase letters.

File: share/goodie/cheat_sheets/json/german.json [hunk 1]
```diff
@@ -14,24 +14,24 @@
       "Getting Help",
       "Travelling",
       "Etiquette",
-      "Dinner on the Town"
+      "Going out for Dinner"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/german.json]: we can lowercase "dinner"

File: share/goodie/cheat_sheets/json/german.json [hunk 2]
```diff
@@ -14,24 +14,24 @@
...
       "Etiquette" : [
          {
-            "val" : "Guten Nacht",
-            "key" : "Good Night"
-         },
-         {
-            "val" : "Guten Mittag",
-            "key" : "Good Afternoon"
+            "val" : "Guten Morgen",
+            "key" : "Good Morning"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/german.json]: lowercase "morning"

File: share/goodie/cheat_sheets/json/german.json [hunk 3]
```diff
@@ -182,7 +178,7 @@
             "key" : "How can I get a taxi?"
          }
       ],
-      "Dinner on the Town" : [
+      "Going out for Dinner" : [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/german.json]: again, lowercase "dinner"

**Comment by moollaza** [comment]
@seisfeld I've left some feedback, once the above is fixed, I'm happy to merge :+1:

**Comment by seisfeld** [comment]
@moollaza Thanks for the feedback. :smile:  Fixed. :+1:

**Comment by moollaza** [comment]
@seisfeld this LGTM :+1:

Thanks again!

**Pull request merged**